README.md

```diff
@@ -0,0 +1,18 @@
+# TDR local AWS services
```

Comment:
Would it be helpful to add an instruction as to how to run the app? Even if its just simply to run the `sbt run` command?

Reply:
Yep - good shout, especially because it's not `sbt run` in this case because it's a multi-project build.